### ISSUE TITLE
black screen problem

### ISSUE THREAD

**Issue by montasser2020** [OneMoreGres/ScreenTranslator#152]
Minutes after using it, I face the problem of the black screen whenever I want to take a capture, I have to restart my computer for this tool to work, I am using Windows 11

**Comment by OneMoreGres** [comment]
It works correctly for some time and then shows a black screen instead of the captured area?

**Comment by Montasser-Hn** [comment]
Yes, I had to restart my computer for it to work again

**Comment by Montasser-Hn** [comment]
I'm sorry, the problem is from me, not from the tool. I use Kaspersky antivirus, When I use the Protected Browser feature, the black screen appears in the tool. I'm sorry. I'll delete this issue after I read it, Thank you🙏🙏

**Comment by Loganavter** [comment]
I have a similar problem, but I'm on arch linux and I don't have any antiviruses

**Comment by OneMoreGres** [comment]
@Loganavter It shows black screen on any window? Or it works for some time?
Regarding translation: deeps is broken, most other translators require translator's window to be open (via tray icons' context menu) or require to solve captcha

**Comment by Loganavter** [comment]
@OneMoreGres the problem is gone, perhaps it is due to the fact that I returned to wayland on xorg. However, I am not sure that the translation display works correctly. In the sense that nothing happens after pressing the button, even in the terminal nothing happens.

**Comment by OneMoreGres** [comment]
Yes, it doesn't work with wayland. There is a test branch with its support, but it's not finished.
To check if something happens, you can enable logging to file option in the settings and then view the log file
